mlflow/spark.py

```diff
@@ -87,7 +137,14 @@ def load_model(path, run_id=None):
     if FLAVOR_NAME not in m.flavors:
         raise Exception("Model does not have {} flavor".format(FLAVOR_NAME))
     conf = m.flavors[FLAVOR_NAME]
-    return PipelineModel.load(os.path.join(path, conf['model_data']))
+    model_path = os.path.join(path, conf['model_data'])
+    dfs_tmp_path = _tmp_path(32)
```

Comment:
Same question about top-level temp directories. Btw, maybe we could just use `uuid` instead of doing randomness ourselves?

Reply:
yeah uuid might make more sense.